src/main/java/controller/MainController.java

```diff
@@ -0,0 +1,29 @@
+package controller;
+
+import controller.game.ChessGameController;
+import controller.room.GameRoomController;
+import controller.user.UserController;
+import database.dao.GameStateDaoImpl;
+import database.dao.PieceDaoImpl;
+import database.dao.RoomDaoImpl;
+import database.dao.UserDaoImpl;
+import dto.RoomDto;
+import dto.UserDto;
+import service.ChessGameService;
+import service.GameRoomService;
+import service.UserService;
+
+public class MainController {
+    private final GameRoomController gameRoomController = new GameRoomController(new GameRoomService(new RoomDaoImpl()));
+    private final ChessGameController chessGameController = new ChessGameController(new ChessGameService(
+            new PieceDaoImpl(), new GameStateDaoImpl()));
```

Comment:
```suggestion
    private final ChessGameController chessGameController
        = new ChessGameController(
            new ChessGameService(
                new PieceDaoImpl(),
                new GameStateDaoImpl()
            )
    );
```
줄바꿈을 활용해서 가독성을 높여보아요!

Reply:
줄바꿈이 젤 어려운 것 같아요... 다양한 고민을 했지만, 이건 생각치 못했네요 😥 어떤 게 가독성이 좋은지 아직 명확하게 판단이 되지 않는 것 같아요.

Reply:
괄호가 많고 파라미터 값들이 많을 때,
같은 깊이에 있는 파라미터들을 한 눈에 보기 좋게 나열하는게 좋은 것 같아요.